### PR TITLE
[#4728] make [Enter] button default to proceed during account creation

### LINKED ISSUE #4728: Make [Enter] button default to proceed
### User Story

As a user I want to user `Enter` key in the application so i dont need to click Confirm / Proceed with a mouse

### Description

*Type*: Bug

*Summary*: Hitting enter does not confirm user's choice , for example when i set up account and want to confirm my selection i need to click `Continue` with a mouse

#### Actual behavior

Can't use enter key to confirm my choice where necessary 

image

### PR DESCRIPTION
**Pull request by rasom**
fix #4728

status: ready

### CODE REVIEW

**Review by vitvly — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/utils/config.cljs [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 ;; CONFIG VALUES
 (def log-level
-  (-> (get-config :LOG_LEVEL "error")
+  (-> (get-config :LOG_LEVEL "debug")
```
Comment (yenda) [inline src/status_im/utils/config.cljs]: default log-level in the absence of env file should stay error IMO why debug ?

Comment (rasom) [reply src/status_im/utils/config.cljs]: yep sorry, that wasn't intended

Comment (rasom) [reply src/status_im/utils/config.cljs]: fixed

**Comment by churik** [comment]
@rasom thanks for PR, it works awesome for IOS, Android and Desktop on account creating.

The same would be great to have for "Add existing account" (mobile, desktop) and for "Start new chat", "Join public chat" (desktop, in mobile is already implemented) to have consistent behavior.

Can it be implemented in this PR as well?

**Comment by rasom** [comment]
ok, done, for "Add existing account" it was added only to password input

**Comment by churik** [comment]
*Tested for IOS, Android:*
- account creation, restoring (can proceed with "Go" key) without tapping on Sign in
- message sending wasn't changed
- can create public chat or enter contact code with "Go" key

*Tested for Desktop:*
- account creation, restoring (can proceed with "Enter" key) without clicking on Sign in
- message sending wasn't changed
- can create public chat or enter contact code with "Enter" key

**Pull request merged**